packages/data/src/lib/dataview/formula/formula.ts

```diff
@@ -0,0 +1,71 @@
+import { CartoError } from '@carto/toolkit-core';
+import { Layer, CARTOSource } from '@carto/toolkit-viz';
+import { DataView } from '../dataview';
+import {
+  AggregationType,
+  aggregate
+} from '../../operations/aggregation/aggregation';
+
+export class FormulaDataView extends DataView {
+  private operation: AggregationType;
+
+  constructor(
+    dataSource: CARTOSource | Layer,
+    column: string,
+    options: FormulaDataViewOptions
+  ) {
+    const { operation } = options || {};
+    validateParameters(operation, column);
+
+    super(dataSource, column);
+
+    this.operation = operation;
+  }
+
+  async getData() {
+    const sourceData = await this.getSourceData([this.column]);
+
+    const filteredValues = sourceData.reduce(
```

Comment:
Hmm, here's one interesting thing. Should we cast strings into numbers? I did that in the CategoryDataView, but not sure if we should or not.

In fact we could make this simpler:

```js
const filteredValues = sourceData
  .map(dataObject => castToNumberOrUndefined(dataObject[this.column]))
  .filter(Boolean)
```

Reply:
Good point...

I would say that we should avoid casting strings, it could be risky, don't you think?. 

I'm gonna try to throw an Error if something not numerical is found in the top features

Reply:
@jesusbotella I've added a small check on the first 10 features, raising an error if finding something different from null or number... After that, assuming that column already has numbers...